lv_conf_template.h

```diff
@@ -12,7 +12,7 @@
  */
 
 /* clang-format off */
-#if 0 /*Set it to "1" to enable content*/
+#if 1 /*Set it to "1" to enable content*/
```

Comment:
It should be `0` by default.

Reply:
what is the point of setting `"LV_CONF_PATH": "${sourceDir}/lv_conf_template.h"`.
But I gues we could set it back to 0 i gues :)